### PR TITLE
Make install proto

### PR DESCRIPTION
**Pull request by miiu96**
Added an extra command in the Makefile in order to install protobuff
This new command `make install-proto`  will download an prepare protobuff dependency in order to can run `go generate ./...`

### CODE REVIEW

File: install-proto.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
+mkdir "${GOPATH}"/src/github.com/gogo
+
```
Comment (iulianpascalau) [inline install-proto.sh:2]: so many empty lines

File: install-proto.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+cd protobuf
+
+git fetch origin pull/659/head:casttypewith
+
+git checkout casttypewith
+
+PROTO_COMPILER=protoc-3.17.3-linux-x86_64.zip
+TEMP_FOLDER_NAME=temp-001
+
+mkdir ~/${TEMP_FOLDER_NAME}
```
Comment (iulianpascalau) [inline install-proto.sh:16]: `mkdir /tmp/${TEMP_FOLDER_NAME}` instead. Also on L18 & L29. Will this make the script fail if the temp folder exists?

File: install-proto.sh [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+git clone https://github.com/gogo/protobuf.git
+
+cd protobuf
+
+git fetch origin pull/659/head:casttypewith
+
+git checkout casttypewith
+
+PROTO_COMPILER=protoc-3.17.3-linux-x86_64.zip
+TEMP_FOLDER_NAME=temp-001
```
Comment (iulianpascalau) [inline install-proto.sh:14]: `temp-proto-001`

**Review by SebastianMarian — APPROVED** [review]

File: install-proto.sh (hunk 1 above)
Comment (miiu96) [reply install-proto.sh:1]: removed.

File: install-proto.sh (hunk 2 above)
Comment (miiu96) [reply install-proto.sh]: refactored.

File: install-proto.sh (hunk 3 above)
Comment (miiu96) [reply install-proto.sh]: done.

**Review by iulianpascalau — APPROVED** [review]

**Review by iulianpascalau — COMMENTED** [review]
Maybe update the readme.md (workflow + makefile run) ?

**Review by SebastianMarian — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**